lib/src/styled_element.dart

```diff
@@ -11,33 +11,33 @@ class StyledElement {
   final List<String> elementClasses;
   List<StyledElement> children;
   Style style;
-  final dom.Node _node;
+  final dom.Node? _node;
 
   StyledElement({
     this.name = "[[No name]]",
-    this.elementId,
-    this.elementClasses,
-    this.children,
-    this.style,
-    dom.Element node,
+    this.elementId = "[[No ID]]",
+    this.elementClasses = const [],
+    required this.children,
+    required this.style,
+    required dom.Element? node,
```

Comment:
Hmm I quickly tried and it's annoying because the `html` package has very little non-nullable claims and also uses the generic `Node` class instead of the specific `Element` which we need. I guess this is fine.

Reply:
I guess this is resolved? Or do any changes need to be made?